### PR TITLE
pkp/pkp-lib#8562 Improve submission wizard after usability tests

### PR DESCRIPTION
**Pull request by NateWr**

### CODE REVIEW

**Review by jonasraoni — CHANGES_REQUESTED** [review]
I didn't see anything besides the duplicated `define('FORM_TITLE_ABSTRACT', 'titleAbstract');`, the rest are just suggestions =]
ps: I didn't download the branch to test the interface.

File: classes/components/forms/publication/Details.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+namespace PKP\components\forms\publication;
+
+use APP\publication\Publication;
+use PKP\components\forms\FieldControlledVocab;
+use PKP\components\forms\publication\TitleAbstractForm;
+use PKP\context\Context;
+use PKP\submission\SubmissionKeywordDAO;
+
+define('FORM_TITLE_ABSTRACT', 'titleAbstract');
```
[the remark this answers is not in the document]
Comment (NateWr) [reply classes/components/forms/publication/Details.php:23]: Good spot! That was extra from a copy/paste. :)

File: classes/components/forms/publication/Details.php [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+        $this->removeField('prefix');
+        $this->removeField('subtitle');
+
+        if (in_array($context->getData('keywords'), [Context::METADATA_REQUEST, Context::METADATA_REQUIRE])) {
+            $this->addField(new FieldControlledVocab('keywords', [
+                'label' => __('common.keywords'),
+                'description' => __('manager.setup.metadata.keywords.description'),
+                'isMultilingual' => true,
+                'apiUrl' => str_replace('__vocab__', SubmissionKeywordDAO::CONTROLLED_VOCAB_SUBMISSION_KEYWORD, $suggestionUrlBase),
```
[the remark this answers is not in the document]
Comment (NateWr) [reply classes/components/forms/publication/Details.php:58]: Using `{` with a string replace complicated the JavaScript a bit, so I used `__abcd__` instead. Happy to revise this in the future but will leave it as-is for now.

File: classes/components/forms/publication/Details.php [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+     * Constructor
+     * 
+     * @param string $suggestionUrlBase The base URL to get suggestions for controlled vocab.
+     */
+    public function __construct(
+        string $action,
+        array $locales,
+        Publication $publication,
+        Context $context,
+        string $suggestionUrlBase,
```
[the remark this answers is not in the document]
Comment (NateWr) [reply classes/components/forms/publication/Details.php:40]: Neat! I didn't know that, thanks!

**Pull request merged**